src/web/src/app/space/link-list/component.html

```diff
@@ -0,0 +1,46 @@
+<ng-container *ngIf="total > 0">
+	<div class="heading">
+		<h2 i18n="@@SPACES.LIST.LINKS.HEADING">Ссылки-приглашения</h2>
+		<button *ngIf="currentSpace" mat-icon-button (click)="onAdd(currentSpace)"><mat-icon>add</mat-icon></button>
+	</div>
+	<div>
+		<!-- form for creating link -->
+		<form *ngIf="!isFormHidden" class="row" [formGroup]="form" (ngSubmit)="onSubmit()">
+			<div *ngIf="!currentSpace">
+				<span>{{formSpaceName}}</span>
+			</div>
+			<div class="fill-width">
+				<mat-form-field appearance="fill">
+					<input matInput type="text" formControlName="name" required />
+					<mat-hint class="red" *ngIf="form.get('name').errors?.['required']" i18n="@@SPACE.LIST.CREATELINK.NAMEREQUIRED">Введите название ссылки</mat-hint>
+					<mat-hint class="red" *ngIf="form.get('name').errors?.['pattern']" i18n="@@SPACE.LIST.CREATELINK.NAMEPATTERN">Допустимые символы: a..z, 0..9 и _</mat-hint>
```

Comment:
Кажется тут и везде в других местах квадратные скобки и кавычки лишние.
```suggestion
					<mat-hint class="red" *ngIf="form.get('name').errors?.required" i18n="@@SPACE.LIST.CREATELINK.NAMEREQUIRED">Введите название ссылки</mat-hint>
					<mat-hint class="red" *ngIf="form.get('name').errors?.pattern" i18n="@@SPACE.LIST.CREATELINK.NAMEPATTERN">Допустимые символы: a..z, 0..9 и _</mat-hint>
```

Reply:
Готово.